src/credential/issuance/04-complete-user-authorization.ts

```diff
@@ -100,13 +111,16 @@ export const completeUserAuthorizationWithQueryMode: CompleteUserAuthorizationWi
         });
     } else {
       // handler for redirectUri
-      Linking.addEventListener("url", ({ url }) => {
+      const sub = Linking.addEventListener("url", ({ url }) => {
```

Comment:
what do you think about renaming `sub` to `eventListener` or something more understandable?